GVFS/GVFS/CommandLine/CacheServerVerb.cs

```diff
@@ -42,6 +42,12 @@ protected override void Execute(GVFSEnlistment enlistment)
 
             using (ITracer tracer = new JsonTracer(GVFSConstants.GVFSEtwProviderName, "CacheVerb"))
             {
+                string authErrorMessage;
+                if (!this.TryAuthenticate(tracer, enlistment, out authErrorMessage))

```

Comment:
Yes before this would end up in `HttpRequestor. SendRequest`, which then called `GitAuthentication.TryGetCredentials`. But with the changes in this PR, that call gets skipped because `IsAnonymous` is still true.

I'm going to fix up the `GitAuthentication` class to make this pattern more explicit. It must be initialized at process startup to attempt an anonymous query and if that fails, request/generate a PAT.